src/patternfly/components/Menu/menu.scss

```diff
@@ -1,252 +1,236 @@
-  --#{$menu}--m-nav__item--Color: var(--#{$pf-global}--Color--light-100);
-  --#{$menu}--m-nav__item--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$menu}--m-nav__item--OutlineOffset: calc(var(--#{$pf-global}--spacer--xs) * -1);
-  --#{$menu}--m-nav__item--before--BorderBottomColor: var(--#{$pf-global}--BackgroundColor--dark-200);
-  --#{$menu}--m-nav__item--before--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$menu}--m-nav__item--hover--after--BorderLeftColor: var(--#{$pf-global}--BorderColor--200);
-  --#{$menu}--m-nav__item--hover--after--BorderLeftWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$menu}--m-nav__item--hover--after--Top: calc(var(--#{$menu}--m-nav__item--before--BorderBottomWidth) * -1);
-  --#{$menu}--m-nav__item__list-item--first-child__item--hover--after--Top: 0;
-  --#{$menu}--m-nav__item-description--Color: var(--#{$pf-global}--Color--light-200);
-
-  // Nav nested menus
-  --#{$menu}--m-nav--c-menu--left-offset: #{pf-size-prem(4px)};
-  --#{$menu}--m-nav--c-menu--m-top--bottom-offset: 0;
-  --#{$menu}--m-nav--c-menu--m-left--right-offset: #{pf-size-prem(4px)};
-  --#{$menu}--m-nav--c-menu--Top: calc(var(--#{$menu}--m-nav__item--before--BorderBottomWidth) * -1);
-  --#{$menu}--m-nav--c-menu--Left: calc(100% - var(--#{$menu}--m-nav--c-menu--left-offset));
-  --#{$menu}--m-nav--c-menu--m-left--Right: calc(100% - var(--#{$menu}--m-nav--c-menu--m-left--right-offset));
-  --#{$menu}--m-nav--c-menu--m-top--Bottom: calc(0 + var(--#{$menu}--m-nav--c-menu--m-top--bottom-offset));
-  --#{$menu}--m-nav__list-item--first-child--c-menu--Top: 0;
-
- inset-block-start: var(--#{$menu}--Top);
-  z-index: var(--#{$menu}--ZIndex);
+// - Menu shared display grid
+.#{$menu},
+.#{$menu}__list,
+.#{$menu}__group {
+  @include pf-v5-hidden-visible(grid);
+}
+
+// - Menu shared display flex
+.#{$menu}__content,
+.#{$menu}__list-item,
+.#{$menu}__item,
+.#{$menu}__item-main,
+.#{$menu}__breadcrumb,
+.#{$menu}__item-check,
+.#{$menu}__item-action,
+.#{$menu}__item-action-icon {
+  @include pf-v5-hidden-visible(flex);
+}
+
+// - Menu shared position relative
+.#{$menu}__list,
+.#{$menu}__list-item,
+.#{$menu}__breadcrumb,
+.#{$menu}__footer {
+  position: relative;
+}
+
+// - Menu disabled
+.#{$menu}__item,
+.#{$menu}__action {
+  &:is(.pf-m-disabled, :disabled, .pf-m-aria-disabled, [aria-disabled="true"]) {
+    --#{$menu}__item--Color: var(--#{$menu}__item--m-disabled--Color);
+    --#{$menu}__item-toggle-icon--Color: var(--#{$menu}--icon--disabled--Color);
+    --#{$menu}__item-external-icon--Color: var(--#{$menu}--icon--disabled--Color);
+    --#{$menu}__item-description--Color: var(--#{$menu}--icon--disabled--Color);
+    --#{$menu}__list-item--BackgroundColor: transparent;
+
+    pointer-events: none;
+  }
+}
+
+// - Menu hidden
+[class*="#{$menu}"]:is([hidden]) {
+  display: none;
+}
+
+// - Menu
+.#{$menu} {
+  // TODO: Explore this fallback method
```

Comment:
I think this fallback method is also up in breadcrumb? Can you revert that back also for now?